App/AppShell.xaml

```diff
@@ -125,7 +125,8 @@
                   x:Name="Header">
                 <Grid.GestureRecognizers>
                     <TapGestureRecognizer Command="{Binding OpenAuth}"
-                                          NumberOfTapsRequired="3"/>
+                                          NumberOfTapsRequired="2"/>
```

Comment:
We can just reduce the number of taps for now. We will remove this very soon anyways